prusti-launch/src/bin/prusti-server.rs

```diff
@@ -70,3 +75,22 @@ fn process(args: Vec<String>) -> Result<(), i32> {
         Err(exit_status.code().unwrap_or(-1))
     }
 }
+
+#[cfg(target_family = "unix")]
+fn sigint_handler() {
+    // Killing the process group terminates the process tree
+    killpg(getpgrp(), Signal::SIGKILL).expect("Error killing process tree.");
+}
+
+#[cfg(target_family = "windows")]
+fn sigint_handler() {
+    // Kill process tree rooted at prusti-server.exe
+    let kill_command: &str = &*format!("TASKKILL /PID {} /T /F", process::id());
+
+    Command::new("cmd")
+        .args(&["/C", kill_command])
```

Comment:
I wonder, why not just spawn `taskkill` with the arguments above, instead of spawning `cmd` first?

Reply:
Both the above suggestions are good. First spawning `cmd` then passing `taskkill` as an argument was an artifact of a previous implementation that I forgot to change. It's now updated.